schema-loader/README.md

```diff
@@ -1,39 +1,47 @@
 # Scalar DB Schema Loader
-This tool creates/deletes Scalar DB schemas (namespaces and tables) based on a provided schema file. Also, it automatically adds the Scalar DB transaction metadata (used in the Consensus Commit protocol) to the tables when you set the `transaction` parameter `true` in the schema file.
 
-There are two ways to specify general CLI options in schema-loader.
+This tool creates/deletes Scalar DB schemas (namespaces and tables) based on a provided schema file.
+Also, it automatically adds the Scalar DB transaction metadata (used in the Consensus Commit protocol) to the tables when you set the `transaction` parameter `true` in the schema file.
+
+There are two ways to specify general CLI options in schema-loader:
   - Pass a Scalar DB configuration file and database-specific options additionally.
   - Pass the options without a Scalar DB configuration.
 
+Note that this tool supports only basic options to create/delete a table.
+If you want to use advanced feature of the database, please alter your table after creating it with this tool.
```

Comment:
```suggestion
If you want to use advanced features of the database, please alter your table after creating it with this tool.
```
since I see `storage-spacific` in the other places in the doc.